mgmt/LocalManager.cc

```diff
@@ -924,19 +924,18 @@ LocalManager::startProxy(const char *onetime_options)
     int i = 0;
     char *options[32], *last, *tok;
     bool open_ports_p = false;
+    char real_proxy_options[16384]; // Arbitrary
```

Comment:
You could use `static const size_t OPTION_SIZE = 16384;` here and then use `OPTION_SIZE` every where later instead of `sizeof(real_proxy_options)`.